src/node/internal/crypto.d.ts

```diff
@@ -1,5 +1,13 @@
 // Copyright (c) 2017-2022 Cloudflare, Inc.
 // Licensed under the Apache 2.0 license found in the LICENSE file or at:
 //     https://opensource.org/licenses/Apache-2.0
+import {
+    Buffer,
+} from 'node-internal:internal_buffer';
+
+// TODO: Export this globally?
+export type arrayLike = ArrayBuffer|SharedArrayBuffer|string | Buffer | DataView;
 
 export function getRandomInt(min: number, max: number): number;
+
+export function getPbkdf(password: arrayLike, salt: arrayLike, iterations: number, keylen: number, digest: String) : ArrayBuffer;
```

Comment:
Nit: new line after.
```suggestion
export function getPbkdf(password: arrayLike, salt: arrayLike, iterations: number, keylen: number, digest: String) : ArrayBuffer;

```